src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="block start--day-wed month--length-31">
+      <div class="block__item"></div>
```

Comment:
I am not sure that naming BEM block as `block` and its elements as `item` is a good idea. Maybe `calendar` and `calendar__day` ?